core/src/main/java/io/grpc/transport/AbstractClientStream.java

```diff
@@ -125,8 +130,8 @@ protected void inboundHeadersReceived(Metadata.Headers headers) {
    *
    * @param frame the received data frame. Its ownership is transferred to this method.
    */
-  protected void inboundDataReceived(ReadableBuffer frame) {
-    Preconditions.checkNotNull(frame, "frame");
```

Comment:
> @ejona86: Java style guide says:
> 
> > You are advised not to comment on the following issues in code reviews:
> > ...
> > Import order. Don't itemize; just use the tools (see Section 9.3, Style-related tools).
> > ...
> > It is reasonable to ask the reviewee to check their editor configuration, but don't point out specific violations.

That is in the _internal_ style guide. We don't have the same tools. Internally we have linters that are very precise for imports and will automatically tell the reviewee of the wrong of their ways. Checkstyle is not as precise for imports. And this instances were causing problems. I think my argument still stands: a change messing up previously-correct formatting when not changing adjacent lines is harmful.

> Code reviews are more of a sanity check, not an exact correctness check. There is a list of best practices, which includes
> > Do not force your personal preferences. If you are suggesting to change something, there should be a clear win. If code complies with style guidelines and is not wrong, then code author is right.

This is fundamentally the same thing I am arguing. A _change_ needs to have a clear win. If it doesn't it shouldn't be made. I feel _you_ are forcing your personal preferences by changing existing code. If it was new code, I wouldn't have limited problem (I may still ask for a description to `checkNotNull()` for public interfaces).

I've not heard any reason why I couldn't make a PR immediately after this one that reverts back to the original form.

> It also stipulates that consistency in the code base should be enforced:
>> Agree as a team to follow specific style where Java Style guide leaves it open-ended and enforce it

> So, this means that the entire file should be changed at once, rather than just in single lines that changed. It either needs to be up to the PR author, or consistently one way across the code, but not neither, and not both. If you don't like the code review noise, then let's pick a hard-and-fast rule and follow it.

I don't comprehend the consistency argument used like you have. How is the removal of "Preconditions." in Http2ClientStream.java for "consistency"? It was consistent before your changes, and you did not add any Preconditions. You simply changed existing code that was already following the style guide to your personal preference.

How is that helpful? If that is permitted (which I assert it shouldn't be), then what would prevent us from just constantly changing it back and forth? What would prevent the use of full-file reformatting (which really can't be allowed)?